dev/automated_tests/pubspec.yaml

```diff
@@ -70,6 +70,6 @@ dependencies:
 flutter:
   uses-material-design: true
   assets:
-  - icon/
+    - icon/test.png
```

Comment:
This was indented wrong and never loaded the asset

Reply:
The previously used asset loader workaround does not throw errors in the same way as the regular asset loader: https://github.com/flutter/flutter/blob/master/packages/flutter_test/lib/src/_binding_io.dart#L34